POLICIES.md

```diff
@@ -228,8 +231,106 @@ spec:
+| `../../../../trust/enabled` | Set as `true` to allow only images that are [signed for content trust](https://cloud.ibm.com/docs/Registry?topic=Registry-registry_trustedcontent) to be deployed. Set as `false` to ignore whether images are signed. |
+| `../../../../trust/signerSecrets/name` | If you want to allow only images that are signed by particular users, specify the Kubernetes secret with the signer name. Omit this field or leave it empty to verify that images are signed without enforcing particular signers. For more information, see [Specifying trusted content signers in custom policies](#specifying-trusted-content-signers-in-custom-policies). |
+| `../../../../va/enabled` | Set as `true` to allow only images that pass the [Vulnerability Advisor](https://cloud.ibm.com/docs/Registry?topic=va-va_index) scan. Set as `false` to ignore the Vulnerability Advisor scan. |
+
+**Table 1**. Understanding the `.yaml` components for the Kubernetes custom resource definition.
+
+Before you begin, [target your `kubectl` CLI](https://cloud.ibm.com/docs/containers?topic=containers-cs_cli_install#cs_cli_configure) to the cluster. Then, complete the following steps:
+
+1. Create a [Kubernetes custom resource definition](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/) `.yaml` file. For more information, see Table 1.
+
+   ```yaml
+   apiVersion: securityenforcement.admission.cloud.ibm.com/v1beta1
+   kind: <ClusterImagePolicy_or_ImagePolicy>
+   metadata:
+     name: <crd_name>
+   spec:
+     repositories:
+       - name: <repository_name>
+         policy:
+         trust:
+             enabled: <true_or_false>
+             signerSecrets:
+             - name: <secret_name>
+         va:
+             enabled: <true_or_false>
+   ```
+
+2. Apply the `.yaml` file to your cluster.
+
+   ```
+   kubectl apply -f <filepath>
+   ```
+   
+### Specifying trusted content signers in custom policies
+
+If you use content trust, you can verify that images are signed by particular signers. Deployment is allowed only if the most recent signed version is signed by all the listed signers. To add a signer to a repository, see [Managing trusted signers](https://cloud.ibm.com/docs/Registry?topic=Registry-registry_trustedcontent#trustedcontent_signers).
+
+To configure the policy to verify that an image is signed by a particular signer:
+
+1. Get the signer name (the name that was used in `docker trust signer add`), and the signer's public key.
+2. Generate a Kubernetes secret with the signer name and their public key.
+
+   ```
+   kubectl create secret generic <secret_name> --from-literal=name=<signer_name> --from-file=publicKey=<key.pub>
+   ```
+
+3. Add the secret to the `signerSecrets` list for the repository in your policy.
+
+   ```yaml
+   - name: example
+     policy:
+       trust:
+         enabled: true
+         signerSecrets:
+         - name: <secret_name>
+   ```
```

Comment:
This section should go under the heading for `trust`, because it's not valid for `simple`.